xpra-client-android/src/main/java/com/github/jksiezni/xpra/MainActivity.java

```diff
@@ -21,36 +21,44 @@
 import android.app.FragmentManager;
 import android.os.Bundle;
 import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
 import android.util.Log;
 
 import com.github.jksiezni.xpra.fragments.ServersListFragment;
 import com.j256.ormlite.android.apptools.OpenHelperManager;
 
-import java.lang.reflect.Constructor;
-import java.lang.reflect.InvocationTargetException;
+//import java.lang.reflect.Constructor;
+//import java.lang.reflect.InvocationTargetException;
+
+//import xpra.protocol.packets.HelloRequest;
+import java.util.Objects;
 
-import xpra.protocol.packets.HelloRequest;
 import xpra.protocol.packets.SetDeflate;
 
 public class MainActivity extends AppCompatActivity implements GlobalActivityAccessor {
 
+	/**
+	 *
+	 */
 	private FloatingActionButton floatingButton;
 
+	public MainActivity() {
+		floatingButton = findViewById(R.id.floatingButton);
```

Comment:
It won't work, because `findViewById()` method must be called after `setContentView()`. The 'onCreate(Bundle savedInstanceState)' method is a proper place to initialize views and other stuff.

Reply:
So just writing `setContentView()` before `floatingButton = findViewById(R.id.floatingButton);` would help?

Reply:
Nope. Read about managing the activity lifecycle, especially `onCreate()` here:
https://developer.android.com/guide/components/activities/intro-activities
and
https://developer.android.com/guide/components/activities/activity-lifecycle